src/backend/metal/src/device.rs

```diff
@@ -703,6 +703,49 @@ impl Device {
 
         arg
     }
+
+    #[cfg(feature = "core_video")]
+    pub unsafe fn create_image_view_with_metal_texture(
+        &self,
+        texture: metal::Texture,
+        kind: image::ViewKind,
+        format: format::Format,
+        swizzle: format::Swizzle,
+        range: image::SubresourceRange,
+    ) -> Result<n::ImageView, image::ViewError> {
+        let mtl_format = match self
+            .shared
+            .private_caps
+            .map_format_with_swizzle(format, swizzle)
+            {
```

Comment:
indentation is off here

Reply:
It's formatted by rustfmt...

Reply:
oh really? that's an unfortunate way to format :(
perhaps, I need to play with our config more